deploy/ansible/playbooks/recreate_databases.yml

```diff
@@ -0,0 +1,14 @@
+---
+# note, horizon must be stopped prior to running this
+- hosts: eco-fed-beta-horizon
```

Comment:
hardcoded node name, rename to "example", this also appears in other places in this pr